### ISSUE TITLE
change auto-suggest priority to prefer more recently active players

### ISSUE THREAD

**Issue by bedwards** [kruser/pitchfx-site#26]
change the priority of the auto-suggest to be something like number of at bats or innings pitches in recent years. more active players are more likely what people are searching for.

**Comment by kruser** [comment]
I like it. I can also get a status (active|free-agent|retired) for each player that we could use to inform this.

**Comment by kruser** [comment]
This is semi-related to issue #33 

With the mugshots now in the search auto-complete it is more obvious who's who. Players that don't have a mugshot on mlb.com are shown with a default grey shadow avatar.

**Comment by albertlyu** [comment]
Given @kruser 's comment in #36 regarding retrieving playerInfo, does that mean we wouldn't want to insert the status of each player (active, free-agent, retired) into the auto-complete either? Or is there another way we can get the status of the player other than from playerInfo.status?

**Comment by kruser** [comment]
I think now that we have mug shots on the auto complete we should close this issue.

I'd be hesitant to alter the sort any further. The search results are best match based on the input right now.

At most I think we could alter the etl to write the date when the player was last found in a game and some how use that. that wouldn't have external dependencies.

I'll leave this open for comment for a little while.

**Comment by kruser** [comment]
My son convinced me that we should do this, so let's leave it open. When you type in just a few characters or a common name, it still alpha sorts as there are so many search results with the same score. So I would suggest we do a double sort...
- score descending
- last regular season game date descending

This will require some work on the atbat-mongodb project to ETL that data into the players collection/table.

This first "francisco" has never played in a regular season game. He likely came in the DB via a spring training game.
![image](https://f.cloud.github.com/assets/2208700/2239330/82f7a02c-9c28-11e3-94e9-d9218a57c90e.png)

**Comment by kruser** [comment]
I added an issue in the ETL project. This is required before I can implement this sort.
https://github.com/kruser/atbat-mongodb/issues/6

**Comment by kruser** [comment]
I updated the ETL so the players collection will be updated on each pull with the start time of the games for the day.

``` BSON
{
    "_id" : ObjectId("53156414826d1a4345003a40"),
    "first" : "Kyle",
    "lastSeen" : ISODate("2014-03-03T02:05:00Z"),
    "last" : "Crick",
    "id" : NumberLong(605195)
}
```

And I updated the search API to sort on lastSeen:descending. Currently on pitchfx.org this value isn't populated. Starting tomorrow morning players that played today will get this value set. Then as we escape spring training the active players will be the only ones with lastSeen dates being more up-to-date. So this will get better after spring.
